bindings/python/python/opendal/__init__.pyi

```diff
@@ -23,7 +23,10 @@ class Operator:
     def __init__(self, scheme: str, **kwargs): ...
     def read(self, path: str) -> bytes: ...
     def open_reader(self, path: str) -> Reader: ...
-    def write(self, path: str, bs: bytes): ...
+    def write(self, path: str, bs: bytes,
+              append: bool=None, buffer: int=None,
```

Comment:
How about accept a `kwargs` here and we can convert to `OpWrite` from `PyDict`?

Reply:
Ok, seems not a good idea. There is no public API to accept an OpWrite.

Reply:
use of `**kwargs` was not my first choice, because I thought accepting a dict would make the parameter names less apparent.

But 7 or 9 parameters is perhaps too many. The current code was rejected by `clappy` in CI.

If a dict seems better I can change this.